lib/os/fdtable.c

```diff
@@ -113,7 +113,7 @@ static int _check_fd(int fd)
 
 	fd = k_array_index_sanitize(fd, ARRAY_SIZE(fdtable));
 
-	if (!atomic_get(&fdtable[fd].refcount)) {
```

Comment:
nit: this changes `fd` to `ufd`, intended?

Reply:
Yes, once validated fd is converted to unsigned ufd and then forgotten.